src/views/schedule/components/Event.module.scss

```diff
@@ -0,0 +1,36 @@
+@import "styles/gallery-theme";
+
+.card {
+	background-color: $dark-chocolate-brown;
+	color: white;
+	margin-bottom: 30px;
+}
+
+.cardTitle {
+	display: flex;
+	justify-content: space-between;
+}
+
+.cardTitleName {
+	color: $orange;
+}
+
+.cardTitleCategory {
+	display: flex;
+	text-align: center;
+	align-items: center;
+	justify-content: center;
+	border-radius: 30px;
+	padding: 0px 20px;
+	background-color: $light-orange;
+	color: $midnight-blue;
+}
```

Comment:
Note: Bootstrap has a [Badge](https://getbootstrap.com/docs/5.3/components/badge/) component we can use